critiquebrainz/frontend/templates/review/entity/base.html

```diff
@@ -88,9 +109,13 @@
           <a id="old-revisions" href="{{ url_for('review.revisions', id=review.id) }}">{{ _('View older revisions') }}</a>
         {% else %}
           <div class="row">
-            <div class="col-md-5">
-              {{ _('<a href="%(link)s">Sign in</a> to rate this review', link=url_for('login.index', next=url_for('review.entity', id=review.id))) }}
-            </div>
+            {% if review.text %}
+              <div class="col-md-5">
+               {{ _('<a href="%(link)s">Sign in</a> to rate this review', link=url_for('login.index', next=url_for('review.entity', id=review.id))) }}
+              </div>
+            {% else %}
+              <div class="col-md-5"></div>
```

Comment:
Why keep this duplicate block definition around? It might be better to just move the condition inside it.